### PR TITLE
Server count and mod count

### PR DESCRIPTION
**Pull request by EstatoDeviato**

### CODE REVIEW

**Comment by Crystalwarrior** [comment]
I'm not sure I like people knowing how many mods are online, especially on KFO since technically any mod can just pop in via mod password, enforce shit, and pop out.
I also want people to use the modcall button even if no mods are online since the modcall button does connect to the Discord.

**Comment by EstatoDeviato** [comment]
If you want, I can remove mod_count

**Review by EstatoDeviato — COMMENTED** [review]
Remove mod_count

File: server/client_manager.py [hunk 1]
```diff
@@ -631,18 +631,20 @@ def set_area(self, area, target_pos=""):
...
                         count = count + 1
                 hub.count = count
+                server_count = server_count + hub.count
             for c in self.server.client_manager.clients:
                 if c.viewing_hub_list:
                     c.send_command(
                         "FA",
                         *[
-                            "{ Hubs }\n Double-Click me to see Areas\n  _______",
+                            f"{self.server.config['masterserver_name']}\n Users Online: {server_count}\n Double-Click me to see Areas\n _______",
```
Comment (Crystalwarrior) [inline server/client_manager.py:647]: Not sure I like this. What if masterserver_name is not defined, or the server is not listed on the master server?

File: server/client_manager.py [hunk 2]
```diff
@@ -1522,18 +1524,20 @@ def remove_client(self, client):
...
                     count = count + 1
             hub.count = count
+            server_count = server_count + hub.count
         for c in self.server.client_manager.clients:
             if c.viewing_hub_list:
                 c.send_command(
                     "FA",
                     *[
-                        "{ Hubs }\n Double-Click me to see Areas\n  _______",
+                        f"{self.server.config['masterserver_name']}\n Users Online: {server_count}\n Double-Click me to see Areas\n _______",
```
Comment (Crystalwarrior) [inline server/client_manager.py:1540]: Not sure I like this. What if masterserver_name is not defined, or the server is not listed on the master server?

**Review by Crystalwarrior — CHANGES_REQUESTED** [review]
I suggest to keep each PR to just 1 specific thing, the change to display server name in area list is separate from the PR's title

**Comment by Crystalwarrior** [comment]
Stale cuz I suck :( reopen if updated

**Pull request closed without merging**